evm/build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
   implementation 'org.hyperledger.besu:arithmetic'
   implementation 'org.hyperledger.besu:bls12-381'
   implementation'org.hyperledger.besu:gnark'
-  implementation 'tech.pegasys:jc-kzg-4844'
+  implementation 'io.consensys.protocols:jc-kzg-4844:2.0.0'
```

Comment:
the version is defined in the `platform` project, so it needs to be removed here